mars/services/task/execution/api.py

```diff
@@ -67,6 +67,10 @@ async def __aexit__(self, exc_type, exc_val, exc_tb):
     async def get_available_band_slots(self) -> Dict[BandType, int]:
         """Get available band slots."""
 
+    @abstractmethod
+    async def get_chunk_bands(self, chunk_keys: List[str]) -> Dict[str, BandType]:
```

Comment:
About if it is reasonable to add this API to execution, what's your opinion? @fyrestone

Reply:
The API is a bit wired for execution. From the title of the PR, I think may be the changes are related to graph construction not the execution?

Reply:
Can you use the chunk to bands info for the graph assigner? https://github.com/mars-project/mars/blob/master/mars/services/task/supervisor/processor.py#L244